recipes/harfbuzz/all/conanfile.py

```diff
@@ -92,48 +95,81 @@ def requirements(self):
-        cmake.definitions["HB_HAVE_GRAPHITE2"] = False
-        cmake.definitions["HB_HAVE_GLIB"] = self.options.with_glib
-        cmake.definitions["HB_HAVE_ICU"] = self.options.with_icu
-        if tools.is_apple_os(self.settings.os):
-            cmake.definitions["HB_HAVE_CORETEXT"] = True
-        elif self.settings.os == "Windows":
-            cmake.definitions["HB_HAVE_GDI"] = self.options.with_gdi
-            cmake.definitions["HB_HAVE_UNISCRIBE"] = self.options.with_uniscribe
-            cmake.definitions["HB_HAVE_DIRECTWRITE"] = self.options.with_directwrite
-        cmake.definitions["HB_BUILD_UTILS"] = False
-        cmake.definitions["HB_BUILD_SUBSET"] = self.options.with_subset
-        cmake.definitions["HB_HAVE_GOBJECT"] = False
-        cmake.definitions["HB_HAVE_INTROSPECTION"] = False
-        # fix for MinGW debug build
-        if self.settings.compiler == "gcc" and self.settings.os == "Windows":
-            cmake.definitions["CMAKE_C_FLAGS"] = "-Wa,-mbig-obj"
-            cmake.definitions["CMAKE_CXX_FLAGS"] = "-Wa,-mbig-obj"
-
-        cmake.configure(build_folder=self._build_subfolder)
-        return cmake
+        get(self, **self.conan_data["sources"][self.version],
+                  destination=self.source_folder, strip_root=True)
+
+    def build_requirements(self):
+        self.tool_requires("meson/0.63.3")
+        if not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
+            self.tool_requires("pkgconf/1.9.3")
 
     def build(self):
-        files.apply_conandata_patches(self)
-        cmake = self._configure_cmake()
-        cmake.build()
+
+
+        apply_conandata_patches(self)
+        meson = Meson(self)
+        meson.configure()
+        meson.build()
 
     def package(self):
-        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
-        cmake = self._configure_cmake()
-        cmake.install()
-        files.rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
-        files.rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        def fix_msvc_libname(remove_lib_prefix=True):
+            """remove lib prefix & change extension to .lib"""
+            if not is_msvc(self):
+                return
+            libdirs = getattr(self.cpp.package, "libdirs")
+            for libdir in libdirs:
+                for ext in [".dll.a", ".dll.lib", ".a"]:
+                    full_folder = os.path.join(self.package_folder, libdir)
+                    for filepath in glob.glob(os.path.join(full_folder, f"*{ext}")):
+                        libname = os.path.basename(filepath)[0:-len(ext)]
+                        if remove_lib_prefix and libname[0:3] == "lib":
+                            libname = libname[3:]
+                        rename(self, filepath, os.path.join(os.path.dirname(filepath), f"{libname}.lib"))
+
+        meson = Meson(self)
+        meson.install()
+        fix_msvc_libname()
```

Comment:
Should [fix_apple_shared_install_name](https://docs.conan.io/en/latest/reference/conanfile/tools/apple.html#conan-tools-apple-fix-apple-shared-install-name) also be used here?